docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+      },
+      {
+        name: 'id',
+        type: 'string',
+        description:
+          'Unique id to identify each ComboBox. Used for [accessibility](#Accessibility) purposes.',
+        required: true,
+      },
+      {
+        name: 'label',
+        type: 'string',
+        description: 'Provide a label to identify the ComboBox field.',
+      },
+      {
+        name: 'disabled',
+        type: 'boolean',
+        defaultValue: false,
+        description:
+          'Set disabled state so ComboBox looks inactive, cannot be interacted with, and actions are not available. If tags are passed, they appeared disabled as well. See [tags](#TYags) variant to learn more.',
+      },
+      {
+        name: 'helperText',
+        type: 'string',
+        description: 'Provides additional information about how to select a ComboBox option.',
+      },
+      {
+        name: 'accessibilityShowButtonLabel',
+        required: true,
+        type: 'string',
+        description: 'Provide feedback when an error on selection occurs.',
+      },
+      {
+        name: 'accessibilityClearButtonLabel',
+        type: 'string',
+        required: true,
+        description: 'Label to describe the icon’s purpose.',
+      },
+      {
+        name: 'errorMessage',
+        type: 'string',
+        description: 'Label to describe the icon’s purpose.',
+      },
+      {
+        name: 'noResultText',
+        type: 'string',
+        required: true,
+        description: 'The text shown when the input value returns no matches',
+      },
+      {
+        name: 'onBlur',
+        type:
+          '({ event: SyntheticFocusEvent<HTMLInputElement> | SyntheticEvent<HTMLInputElement> , value: string }) => void',
+        description: 'Callback when you focus outside the component ',
+      },
+      {
+        name: 'onChange',
+        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        description: 'Callback when user types into the control input field',
+      },
+      {
+        name: 'onClear',
+        type: '() => void',
+        description: 'Callback when user clicks on clear button',
+      },
+      {
+        name: 'onFocus',
+        type: '({ event: SyntheticFocusEvent<>, value: string }) => void',
+        description: 'Callback when you focus on the component',
+      },
+      {
+        name: 'onKeyDown',
+        type: '({ event: SyntheticKeyboardEvent<HTMLTextAreaElement>, value: string }) => void',
+        description: 'Callback for key stroke events. See [tags](#TYags) variant to learn more.',
+      },
+      {
+        name: 'onSelect',
+        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        description: 'Callback when an item is selected',
+      },
+      {
+        name: 'placeholder',
+        type: 'string',
+        description: 'Specify a short description suggestion the expected input for the field',
```

Comment:
Specify a short description that suggests the expected input for the field

Reply:
DONE